libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/Dialogs/Microsoft.AdaptiveSkillDialog.schema

```diff
@@ -0,0 +1,76 @@
+{
+    "$schema": "https://raw.githubusercontent.com/microsoft/botbuilder-dotnet/master/schemas/component.schema",
+    "$role": "union(Microsoft.IDialog)",
+    "title": "Begin a skill dialog",
+    "description": "Begin a remote skill dialog.",
+    "type": "object",
+    "properties": {
+        "id": {
+            "type": "string",
+            "title": "Id",
+            "description": "Optional id for the skill dialog"
+        },
+        "disabled": {
+            "$role": "expression",
+            "type": "boolean",
+            "title": "Disabled",
+            "description": "Optional condition which if true will disable this action.",
+            "examples": [
+                "user.age > 3"
+            ]
+        },
+        "activityProcessed": {
+            "$role": "expression",
+            "type": "boolean",
+            "title": "Activity Processed",
+            "description": "When set to false, the dialog that is called can process the current activity.",
+            "default": true
+        },
+        "resultProperty": {
+            "$role": "expression",
+            "type": "string",
+            "title": "Property",
+            "description": "Property to store any value returned by the dialog that is called.",
+            "examples": [
+                "dialog.userName"
+            ]
+        },
+        "botId": {
+            "$role": "expression",
+            "type": "string",
+            "title": "Skill host bot ID",
+            "description": "The Microsoft App ID that will be calling the skill.",
+            "default": "=settings.MicrosoftAppId"
+        },
+        "skillHostEndpoint": {
+            "$role": "expression",
+            "type": "string",
+            "title": "Skill host",
+            "description": "The callback Url for the skill host.",
+            "default": "=settings.skillHostEndpoint",
+            "examples": [
+                "https://mybot.contoso.com/api/skills/"
+            ]
+        },
+        "skillAppId": {
+            "$role": "expression",
+            "type": "string",
+            "title": "Skill App ID",
+            "description": "The Microsoft App ID for the skill."
+        },
+        "skillEndpoint": {
+            "$role": "expression",
+            "type": "string",
+            "title": "Skill endpoint ",
+            "description": "The /api/messages endpoint for the skill.",
+            "examples": [
+                "https://myskill.contoso.com/api/messages/"
+            ]
+        },
+        "cardNoMatchResponse ": {
```

Comment:
is this the correct name? What does it do?

Reply:
Fixed (it was left over copy/paste)